packages/gafl-webapp-service/src/pages/order-complete/route.js

```diff
@@ -29,6 +29,7 @@ const getData = async request => {
   }
 
   await request.cache().helpers.status.set({ [COMPLETION_STATUS.completed]: true })
+  await request.cache().helpers.status.setCurrentPermission({ currentPage: ORDER_COMPLETE.page })
```

Comment:
I saw the new tests above, which is fine, however I'd like to see a direct test of this. If I was to delete this line would any tests fail?